cli/src/installers/envVars.ts

```diff
@@ -37,8 +37,9 @@ DATABASE_URL=file:./db.sqlite
     envContent += `
 # Next Auth
 # You can generate the secret via 'openssl rand -base64 32' on Linux
+# Make sure not to keep the secret empty
 # More info: https://next-auth.js.org/configuration/options#secret
-NEXTAUTH_SECRET=
+NEXTAUTH_SECRET=CHANGE_ME
```

Comment:
I like this much more than the secret in the other NEXTAUT_SECRET related pr that we merged, because it makes it clear that this shouldn't be used as the "real" value. Please keep your version when resolving the merge conflict.